rules/combo/stealer/wallet.yara

```diff
@@ -24,6 +24,13 @@ rule crypto_stealer : critical {
     $w_trezor = "Trezor"
     $w_exodus = "Exodus"
     $w_coinomi = "Coinomi"
+    $not_cats = /\"cats\": \[[^]]{0,64}/
```

Comment:
This isn't an exhaustive representation of the Wapalyzer schema but these should be broad enough to match content from those technology files.